src/styles/components/_home.scss

```diff
@@ -49,7 +49,7 @@ $breakpoint-top-section-scale-up: 70rem;
       h1 {
         font-size: map.get($font-sizes, "mobile-heading");
         font-weight: 400;
-        line-height: 1.21; // 34px/28
+        line-height: map.get($line-heights, "lg"); // approx. 34px/24
```

Comment:
Oh yeah, this is a lot better than having a ton of wacky line heights! I'm tweaking the footer on a different branch and having a standard value to work around will simplify things.